feature-libs/product-multi-dimensional/selector/components/selector/product-multi-dimensional-selector.component.html

```diff
@@ -0,0 +1,39 @@
+<ng-container *ngIf="product$ | async as product">
+  <div class="variant-generic-selector">
+    <ng-container *ngFor="let category of categories">
+      <label>{{ category.name }}</label>
+
+      <div
+        *ngIf="category.hasImages; else noImages"
+        class="image-variant-container"
+      >
+        <cx-media
+          *ngFor="let option of category.variantOptions"
+          (click)="changeVariant(option.code)"
+          [container]="option.images"
+          [ngClass]="{ active: option?.code === product.code }"
+          [title]="option.value"
+          class="image-variant"
+        >
+        </cx-media>
```

Comment:
```suggestion
        <cx-media
          *ngFor="let option of category.variantOptions"
          (click)="changeVariant(option.code)"
          [container]="option.images"
          [ngClass]="{ active: option?.code === product.code }"
          [title]="option.value"
          class="image-variant"
        />
```
`minor:` Let's use a self-closing component tag to be in line with SPA coding guidelines. See: https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-Useself-closingcomponenttags

Reply:
fixed